picard/ui/options/general.py

```diff
@@ -41,6 +41,7 @@ class GeneralOptionsPage(OptionsPage):
         config.TextOption("persist", "oauth_refresh_token", ""),
         config.BoolOption("setting", "analyze_new_files", False),
         config.BoolOption("setting", "ignore_file_mbids", False),
+        config.BoolOption("setting", "cluster_new_files", False),
```

Comment:
Could we set it to True, but use an upgrade function to set it to false for existing users?